integration-tests/src/test/java/org/springframework/aot/test/SampleReflection.java

```diff
@@ -16,25 +16,19 @@
 
 package org.springframework.aot.test;
 
-import java.lang.reflect.Method;
-
 /**
  * @author Brian Clozel
  * @since 6.0
  */
 public class SampleReflection {
 
-	@SuppressWarnings("unused")
 	public void sample() {
-		String value = "Sample";
-		Method[] methods = String.class.getMethods();
+		String.class.getMethods();
 	}
 
-	@SuppressWarnings("unused")
 	public void multipleCalls() {
-		String value = "Sample";
-		Method[] methods = String.class.getMethods();
-		methods = Integer.class.getMethods();
+		String.class.getMethods();
```

Comment:
not sure if the test is still working as expected, as I could not execute it locally.